elastic/kibana
# [Infra] Disable Top 10 functions full screen table in a flyout

## Summary

This PR disables functions full-screen option in the asset details flyout. It adds control of the `showFullScreenSelector` to the `EmbeddableFunctionsGrid` so we can set it based on the render mode in the asset details - in apm it will be set to true as before.

## Testing

- Go to the asset details page
- Check the Universal Profiling tab > Top 10 Functions tab
   - The full-screen option should be visible: 
   ![image](https://github.com/user-attachments/assets/aace7ca0-ed4f-404d-8cbf-91c29088c133)

   - Same in the tab inside APM service view:
    ![image](https://github.com/user-attachments/assets/f885630a-4f43-4c23-9ff4-05c03f7ede72)

- Go to Hosts view and open the flyout for a host 
   - The full-screen option should not be visible: 
    ![image](https://github.com/user-attachments/assets/e724c569-df7a-4e4a-b6c3-9dcfa7b60ad6)
